examples/Makefile

```diff
@@ -4,7 +4,7 @@ run:
 	chmod 777 log
 	docker-compose build
 	docker-compose up -d service
-	sleep 10 # wait for bootstrap
+	sleep 30 # wait for bootstrap
```

Comment:
Almost saying that it should ping or health check the service per earlier conversation on other PR.

Reply:
I want to address it in another PR 😉